compiler/src/iree/compiler/Dialect/LinalgExt/Transforms/AggregatedOpInterfaceImpl.cpp

```diff
@@ -346,36 +353,25 @@ OnlineAttentionOp::decomposeOperation(OpBuilder &b) {
             return sizes[cast<AffineDimExpr>(dimExpr).getPosition()];
           }));
 
-      // We rescale `p` to use the full range and pass back the `pScale`.
-      Value maxEmpty = b.create<tensor::EmptyOp>(loc, mSizes, reduceType);
```

Comment:
Do we not need to update test for this?

Reply:
The problem with `fp8` is that testint hardware is extremely limited. I could make a lit test but that would boil down to being a change test.

Reply:
I see I see, I'm not going to block since you've locally validated, plus it is not failing our other non fp8 attention, and I think we need it on the nightly for progress. 

but yeah a lint test would be nice for the FP8 since we are playing around with the decomp.